algo.js

```diff
@@ -508,10 +543,25 @@ if (sendMail === "send") {
 // Run the Algorithm
 const mode = process.argv[2];
 if (mode === 'run') {
-  addMatches('Matches (Sep 12th)', ['Matches (May 2nd)', 'Matches (May 15th)', 'Matches (June 2nd)', 'Matches (June 13th)', 'Matches (June 20th)', 'Matches (June 27th)', 'Matches (July 6th)', 'Matches (July 11th)', 'Matches (July 19th)', 'Matches (July 25th)', 'Matches (Aug 1st)', 'Matches (Aug 8th)', 'Matches (Aug 15th)', 'Matches (Aug 22nd)', 'Matches (Aug 30th)', 'Matches (Sep 5th)'])
+  addMatches('Matches (Sep 19th)', jsonObj);
```

Comment:
Instead of hardcoding the date, we could just have the system get the date using Javascript's built-in date functions

Reply:
will do